config/default/crds/serving_v1alpha1_kfservice.yaml

```diff
@@ -65,6 +65,19 @@ spec:
                     0 in case of no traffic
                   format: int64
                   type: integer
+                onnx:
```

Comment:
add ONNX image in https://github.com/kubeflow/kfserving/blob/master/config/default/configmap/kfservice.yaml